common/airgapped/patch-delete-secret-ref.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: not-used-in-a-patch
+spec:
+  secretRef:
+    $patch: delete
```

Comment:
Looks like `$patch: delete` instruction breaks the `Check licenses.d2iq.yaml` check: https://github.com/mesosphere/kommander-applications/actions/runs/8018555753/job/21904680915?pr=1969